GT-SGA/JacketPages
# Move View All Bills into React

Tracks #53, #47, #29 

### Testing
* Open two separate terminal tabs and run yarn server in one and yarn client in the other.
* Go to the view all bills tab in the nav bar
* All of the bills should be displayed on one long page with some CSS issues which can be fixed later.
